src/ProjectCracker/ProjectCracker.fs

```diff
@@ -347,6 +351,7 @@ let private projectTarget(csproj: FileInfo) =
     let projectAssetsJson = FileInfo(Path.Combine [|csproj.DirectoryName; "obj"; "project.assets.json"|])
     if projectAssetsJson.Exists then 
         let assets = parseProjectAssets(projectAssetsJson)
+        let dllName = assets.projectName + ".dll"
```

Comment:
I'd like feedback specially on `projectTarget`. I did the minimum amount of changes I could overall, but I wonder if this function needs cleanup.

Reply:
Hmm, I'm not really sure what you mean. There's an assumption in this function that the .dll is always in `{projectDir}/bin/debug/{framework}/{projectName}.dll`, and there's probably a bunch of other cases where this rule needs to be more complicated...not really sure that's "cleanup" though.

Reply:
Oh I see, two dllNames